test-cases/longevity/longevity-encryption-at-rest-200GB-6h.yaml

```diff
@@ -0,0 +1,31 @@
+test_duration: 380
+prepare_write_cmd: "cassandra-stress write cl=ALL n=200200300  -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..200200300 -log interval=15"
```

Comment:
Let's make it a little bit different than others, so we can test more things + maybe catch issues.
please change the -col 'size=FIXED(1024 n=FIXED(1)' to:
-col 'size=FIXED(12) n=FIXED(100)'

(change in all places)